src/tools/compiletest/src/lib.rs

```diff
@@ -163,7 +163,8 @@ pub fn parse_config(args: Vec<String>) -> Config {
         )
         .optopt("", "edition", "default Rust edition", "EDITION")
         .reqopt("", "git-repository", "name of the git repository", "ORG/REPO")
-        .reqopt("", "nightly-branch", "name of the git branch for nightly", "BRANCH");
+        .reqopt("", "nightly-branch", "name of the git branch for nightly", "BRANCH")
+        .reqopt("", "git-merge-commit-email", "email address used for the merge commits", "EMAIL");
```

Comment:
"the" here should be removed.

Also, this is used to *find* merge commits, not to create them, right? It should specifically be the bors email? That's not clear at all currently, neither here nor in the code.